packages/rum/src/domain/record/observers.ts

```diff
@@ -348,33 +350,61 @@ export function initInputObserver(cb: InputCallback, defaultPrivacyLevel: Defaul
   }
 }
 
-function initStyleSheetObserver(cb: StyleSheetRuleCallback): ListenerHandler {
-  const { stop: restoreInsertRule } = instrumentMethodAndCallOriginal(CSSStyleSheet.prototype, 'insertRule', {
-    before(rule, index) {
-      if (hasSerializedNode(this.ownerNode!)) {
-        cb({
-          id: getSerializedNodeId(this.ownerNode),
-          adds: [{ rule, index }],
-        })
-      }
-    },
-  })
-
-  const { stop: restoreDeleteRule } = instrumentMethodAndCallOriginal(CSSStyleSheet.prototype, 'deleteRule', {
-    before(index) {
-      if (hasSerializedNode(this.ownerNode!)) {
-        cb({
-          id: getSerializedNodeId(this.ownerNode),
-          removes: [{ index }],
-        })
-      }
-    },
-  })
+export function initStyleSheetObserver(cb: StyleSheetRuleCallback): ListenerHandler {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏since we now observe other APIs than stylesheet, what about renaming this method to something more general, like `initCSSObservers`?